tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md

```diff
@@ -28,116 +28,117 @@ The period for idle time for Factory accounts is different than for trial accoun
 
 [ACCORDION-BEGIN [Step 1: ](Create new SAP Fiori project)]
 
-1. In the **Welcome Page** click **Create project from a template**.
+1. In the **Welcome Page** click **Start from template**.
 
-    !![Open Dev Space](03-01-02 AppStudio Welcome Tab withOUT Extensions Loaded Notification_.jpg)
+    !![Open Dev Space](AppStudio Welcome Tab withOUT Extensions Loaded Notification_.jpg)
 
-    >If the Welcome Page does not appear, in the menu bar, select **View | Find Command** to open the **command palette** and select the command **SAP Business Application Studio: Welcome Page**. The command palette is opened at the top-center of the SAP Business Application Studio window.
+    >If the Welcome Page does not appear, in the menu bar, select **View | Find Command** to open the **command palette** and select the command **Welcome**. The command palette is opened at the top-center of the SAP Business Application Studio window.
 
-    >!![Welcome Page from command palette](03-01-03 AppStudio Welcome Page from Command Palette_.jpg)
+    >!![Welcome Page from command palette](AppStudio Welcome Page from Command Palette_.jpg)
```

Comment:
Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

Reply:
done